src/pages/home/_trade-types.tsx

```diff
@@ -19,109 +12,111 @@ import { useLangDirection } from 'components/hooks/use-lang-direction'
 
 type TradeTypesProps = {
     class_name: string
-    image_url: string
-    image_alt: ReactElement
     header: ReactElement
     desc: ReactElement
     link: string
     link_text: ReactElement
     alt: string
+    imageElement: ReactElement
 }
 
 type TradeItemsProps = {
     items_details: TradeTypesProps
 }
 
-const query = graphql`
-    query {
-        trade_type_cfds: file(relativePath: { eq: "home/trade_type_cfds.png" }) {
-            ...fadeIn
-        }
-        trade_type_cfds_eu: file(relativePath: { eq: "home/trade_type_cfds_eu.png" }) {
-            ...fadeIn
-        }
-        trade_type_digitaloptions: file(
-            relativePath: { eq: "home/trade_type_digitaloptions.png" }
-        ) {
-            ...fadeIn
-        }
-        trade_type_multipliers: file(relativePath: { eq: "home/trade_type_multipliers.png" }) {
-            ...fadeIn
-        }
-        trade_type_multipliers_eu: file(
-            relativePath: { eq: "home/trade_type_multipliers_eu.png" }
-        ) {
-            ...fadeIn
-        }
-        trade_type_spreads: file(relativePath: { eq: "home/trade_type_spreads.png" }) {
-            ...fadeIn
-        }
-    }
-`
-
 const items_details_row: TradeTypesProps[] = [
     {
         class_name: 'cfds',
-        image_url: 'trade_type_cfds',
-        image_alt: <Localize translate_text="CFDs" />,
         header: <Localize translate_text="CFDs" />,
         desc: (
             <Localize translate_text="Trade with leverage and low spreads for better returns on successful trades." />
         ),
         link: '/trade-types/cfds/',
         link_text: <Localize translate_text="More on CFDs" />,
         alt: 'cfd',
+        imageElement: (
+            <StaticImage
+                src="../../images/common/home/trade_type_cfds.png"
+                alt="cfd"
+                loading="eager"
+                placeholder="none"
```

Comment:
Do we need `placeholder` in here? I think this prop is optional and we don't need that.

Reply:
Yes, placeholder is optional. if we don't provide it, the default placeholder `"dominantColor"` will be taken and until the image loads, a solid dark background image will displayed.
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#placeholder